requirements.txt

```diff
@@ -1,6 +1,6 @@
 python-dotenv
 prefect==2.19.1
-prefect_dask
+prefect[dask]
```

Comment:
The `[package]` is the standard way to specify an optional dependency of a package: https://setuptools.pypa.io/en/latest/userguide/dependency_management.html#optional-dependencies

So two line of "prefect" requirements are not needed just: pefect[dask]==2.19.1` should suffice.

Reply:
ok... huh. Thanks Brad